wsd/DocumentBroker.hpp

```diff
@@ -433,6 +433,64 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
     std::string _jailId;
     std::string _filename;
 
+    /// The state of the document.
+    /// This regulates all other primary operations.
+    class DocumentState
+    {
+    public:
+        /// Strictly speaking, these are phases that are directional.
+        /// A document starts as New and progresses towards Unloaded.
+        /// Upon error, intermediary states may be skipped.
+        enum class Status
+        {
+            New, //< Doesn't exist, pending downloading.
```

Comment:
I wonder if None instead of New, Destroying instead of Unloading and Destroyed instead of Unloaded wouldn't be a bit more clear (?)

Reply:
Yes, those were the other options. Happy to adopt them.